src/test/checks_run.sh

```diff
@@ -109,6 +109,8 @@ if test "$COVERAGE" = "t"; then
 	include = $(pwd)/src/*
 	parallel = True
 	relative_files = True
+        [report]
+        omit = */flux/utils/*
```

Comment:
Nit: Looks like maybe the previous lines are tabs and these are spaces.  Hard to tell for sure in the Github UI.

Reply:
Good catch, that was the reason coverage broke for python. Already addressed.